tensorflow_federated/python/research/utils/checkpoint_manager.py

```diff
@@ -221,5 +221,5 @@ def _round_num(self, checkpoint_path: str) -> int:
 
   def _get_all_checkpoint_paths(self) -> List[str]:
     """Returns all the checkpoint paths managed by the instance."""
-    pattern = os.path.join(self._root_dir, '{}*'.format(self._prefix))
-    return tf.io.gfile.glob(pattern)
+    root_dir_entries = tf.io.gfile.listdir(self._root_dir)
+    return [os.path.join(self._root_dir, e) for e in root_dir_entries if e.startswith(self._prefix)]
```

Comment:
I suspect this list comprehension needs to be formatted to be less than 80-characters (https://github.com/google/styleguide/blob/gh-pages/pyguide.md#32-line-length)

Could you run `yapf` and `pylint` on this PR? https://github.com/tensorflow/federated/blob/master/CONTRIBUTING.md#coding-style

Reply:
done 
see also https://github.com/tensorflow/federated/pull/908